gtest-parallel

```diff
@@ -281,6 +290,14 @@ elif options.format == 'filter':
 else:
   sys.exit("Unknown output format: " + options.format)
 
+if options.shard_count < 1:
+  sys.exit("Invalid number of shards: %d. Must be at least 1." %
+           options.shard_count)
+if options.shard_index < 0 or options.shard_count <= options.shard_index:
```

Comment:
I don't think so.
For example, if options.shard_count = 1 and options.shard_index = -1, then options.shard_count <= options.shard_index (1 <= -1) would be false.